src/main/resources/api/service_openapi.yaml

```diff
@@ -336,6 +382,22 @@ components:
       schema:
         $ref: '#/components/schemas/Name'
 
+    Role:
+      name: role
+      in: path
+      description: An IAM role
+      required: true
+      schema:
+        $ref: '#/components/schemas/IamRole'
+
+    MemberEmail:
+      name: memberEmail
+      in: path
+      description: A user or group's email. Used for adding or removing IAM permissions
```

Comment:
Validation? Or you count on Sam for that?

Reply:
I was a little split on this one, though right now I just rely on Sam for that. Passing around a user-provided and unvalidated string doesn't feel great, but validating in two places introduces the risk of disagreeing on what's considered a valid email address (e.g. Sam has some list of blocked domains).

Reply:
My concern would not be about the email itself. It would be someone injecting additional URL elements in their address that would change the Sam semantics. So making sure the string does contain a `@` and doesn't contain `?` or `/` or other special URL characters. I'm happy to have Sam deal with whether the email is actually reasonable.

Reply:
Makes sense. Once we start checking for `@` and excluding some special characters (but e.g. not `+`) though I think it makes sense to just use the same regex as Sam, with a comment indicating that's where it came from.

Reply:
Also worth noting that unit tests and connected tests don't exercise the controller, so this sort of validation would be tested in a client test. I've added that to my backlog of tests to write